src/widgets/latest-blocks/LatestBlocks.vue

```diff
@@ -47,33 +47,33 @@ import BaseLink from '@/shared/ui/components/BaseLink.vue';
 import BaseButton from '@/shared/ui/components/BaseButton.vue';
 import BaseContentBlock from '@/shared/ui/components/BaseContentBlock.vue';
 import BaseLoading from '@/shared/ui/components/BaseLoading.vue';
-import type { Block } from '@/shared/api/schemas';
-import { onMounted, ref, shallowRef } from 'vue';
-import { useErrorHandlers } from '@/shared/ui/composables/useErrorHandlers';
+import { computed, watch } from 'vue';
 import TimeStamp from '@/shared/ui/components/TimeStamp.vue';
+import { useParamScope } from '@vue-kakuyaku/core';
+import { handleParamScope } from '@/shared/api/handle-param-scope';
 
 const emit = defineEmits<{
   loaded: [number]
 }>();
 
-const blocks = shallowRef<Block[]>([]);
-const isLoading = ref(false);
-
-const { handleUnknownError } = useErrorHandlers();
-
-onMounted(async () => {
-  try {
-    isLoading.value = true;
-
-    const res = await http.fetchBlocks();
-
-    blocks.value = res.items;
-    emit('loaded', res.pagination.total_items);
-  } catch (error) {
-    handleUnknownError(error);
-  } finally {
-    isLoading.value = false;
-  }
+const scope = useParamScope(
+  () => {
+    return {
+      key: 'latest-blocks',
+      payload: {
+        per_page: 10,
+      },
+    };
+  },
+  ({ payload }) => handleParamScope(payload, http.fetchBlocks)
+);
```

Comment:
The point of setting up a scope is that you have independent virtual reactive environments for each set of parameters. Thus, if parameters change, you don't need to set up various watchers to stop current fetching and start a new one _manually_ - the scope is simply disposed (as if it was a separate component) and a new one is set up.

In this case payload is constant, thus it's pointless to set up a parametrised scope. It only introduces extra complexity.